server.js

```diff
@@ -7715,6 +7715,53 @@ camp.route(/^\/nsp\/npm\/(?:@([^/]+)?\/)?([^/]+)?(?:\/([^/]+)?)?\.(svg|png|gif|j
   }
 }));
 
+// bundle size for npm packages
+camp.route(/^\/bundlephobia\/(min|gzip)\/(?:@([^/]+)?\/)?([^/]+)?(?:\/([^/]+)?)?\.(svg|png|gif|jpg|json)?$/,
```

Comment:
Minified and gzipped aren't exclusive. Does gzip mean minified and gzipped?

Reply:
Yes, it does.

Reply:
Then how about we make it `min` and `min-gzip`?

Reply:
Another commonly used term for this is "minzip"

Reply:
Sure, that works and seems clear.

Reply:
Done.